### PR TITLE
Fix `sled_multi_threaded` example code with threads of `INSERT` and `SELECT`,

### PR DESCRIPTION
**Pull request by ever0de**
### Detail

1. You can check whether the example code can be executed in github actions.
2. Fix the broken sled example code after multiple query string is supported.

### CODE REVIEW

**Comment by ever0de** [comment]
<https://github.com/gluesql/gluesql/runs/6641599693?check_suite_focus=true>
```shell
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: StorageMsg("database is locked")', examples/sled_multi_threaded.rs:38:33
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at 'Something went wrong in the world thread: Any { .. }', examples/sled_multi_threaded.rs:43:14
```

Did you not support multithreading in sled storage? Or maybe the way you use it has changed?

**Comment by ever0de** [comment]
As a result of checking, this code seems to be a code that has not been running before.
The `INSERT` statement does not work at the same time, and the ` txid != lock_txid` condition is applied, so an error message related to `database is locked` appears.

Modify the example code with threads of `INSERT` and `SELECT`.

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -39,3 +39,11 @@ jobs:
...
       - name: Run tests with all features
         run: cargo test --all-features --all-targets --verbose
+      - name: Run hello_world example code
+        run: cargo run --package gluesql --example hello_world
+      - name: Run api_usage example code
+        run: cargo run --package gluesql --example api_usage
+      - name: Run sled_multi_threaded example code
+        run: cargo run --package gluesql --example sled_multi_threaded
+      - name: Run using_config example code
+        run: cargo run --package gluesql --example using_config
```
Comment (panarch) [inline .github/workflows/rust.yml:49]: 👍

**Review by panarch — APPROVED** [review]
Awesome! root examples are now back to be maintained by our rust github action.
Thanks a lot 👍

**Pull request merged**